README.md

```diff
@@ -245,6 +246,39 @@ You can populate below template according to your requirements and use it as you
 
 # WEBDAV_URL_INSECURE="true"
 
+# You can also backup files to any SSH server:
+
+# The URL of the remote SSH server
+
+# SSH_HOST_NAME="server.local"
+
+# The port of the remote SSH server
+# Optional variable default value is `22`
+
+# SSH_PORT=2222
+
+# The Directory to place the backups to on the SSH server.
+
+# SSH_REMOTE_PATH="/my/directory/"
+
+# The username for the SSH server
+
+# SSH_USER="user"
+
+# The password for the SSH server
+
+# SSH_PASSWORD="password"
+
+# The private key path in container for SSH server
+# Default value: /root/.ssh/id_rsa
+# If file is mounted to /root/.ssh/id_rsa path it will be used.
+
+# SSH_IDENTITY_FILE="/root/.ssh/id_rsa"
+
+# The passphrase for the identity file
+
+# SSH_PASSPHRASE="pass"
```

Comment:
Just an idea: to reduce possible mixups of password and passphrase, how about naming this `SSH_IDENTITY_PASSPHRASE` or `SSH_KEY_PASSPHRASE`?

Reply:
Changed to SSH_IDENTITY_PASSPHRASE and test updated with passphrase